### PR TITLE
update readme with new nix substituters

### PR DESCRIPTION
**Pull request by JaredCorduan**

### CODE REVIEW

**Comment by lehins** [comment]
Related to this. NixOS has deprecated `nix.binaryCaches` and`nix.binaryCachePublicKeys`, so we should update the NixOS settings too:

```nix
  nix.settings = {
    substituters = [
      "https://cache.nixos.org"
      "https://cache.iog.io"
    ];
    trusted-public-keys = [
      "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ="
    ];
  };
```

**Comment by JaredCorduan** [comment]
> ```nix
> cache.iog.org
> ```

I think you meant `cache.iog.io`.

**Comment by lehins** [comment]
> I think you meant cache.iog.io.

:smile: Yeah, that's exactly what I meant. I updated a part of the wrong url :man_facepalming:

**Comment by lehins** [comment]
Does the key needs to be updated too?
```diff
-trusted-public-keys = hydra.iohk.io:f/Ea+s+...
+trusted-public-keys = cache.iog.io:f/Ea+s+...
```

**Comment by JaredCorduan** [comment]
> Does the key needs to be updated too?
> 
> ```diff
> -trusted-public-keys = hydra.iohk.io:f/Ea+s+...
> +trusted-public-keys = cache.iog.io:f/Ea+s+...
> ```

I believe that @disassembler told me that they do not.

**Review by michaelpj — APPROVED** [review]
Looks right!

**Review by lehins — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -73,22 +73,23 @@ If you are using [NixOS](https://nixos.org/) add the snippet below to your
...
-nix.binaryCaches = [
-  "https://cache.nixos.org"
-  "https://hydra.iohk.io"
-];
-
-nix.binaryCachePublicKeys = [
-  "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ="
-];
+nix.settings = {
+  substituters = [
```
Comment (disassembler) [inline README.md:78]:
```suggestion
  experimental-features = [ "nix-command" "flakes" ];
  substituters = [
```

File: README.md [hunk 2]
```diff
@@ -73,22 +73,23 @@ If you are using [NixOS](https://nixos.org/) add the snippet below to your
...
+};
 ```
 
 If you are using the `nix` package manager next to another operating system put
 the following in `/etc/nix/nix.conf` if you have a system-wide `nix`
 installation , or in `~/.config/nix/nix.conf` if you have a local installation:
 
 ```
-substituters        = https://hydra.iohk.io https://cache.nixos.org/
+substituters        = https://cache.iog.io https://cache.nixos.org/
```
Comment (disassembler) [inline README.md:93]:
```suggestion
experimental-features = nix-command flakes
substituters        = https://cache.iog.io https://cache.nixos.org/
```

File: README.md [hunk 3]
```diff
@@ -73,22 +73,23 @@ If you are using [NixOS](https://nixos.org/) add the snippet below to your
...
+    "https://cache.iog.io"
+  ];
+  trusted-public-keys = [
+    "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ="
+  ];
+};
 ```
 
 If you are using the `nix` package manager next to another operating system put
 the following in `/etc/nix/nix.conf` if you have a system-wide `nix`
```
Comment (disassembler) [inline README.md]:
```suggestion
the following in `/etc/nix/nix.conf`:
```

File: README.md [hunk 4]
```diff
@@ -73,22 +73,23 @@ If you are using [NixOS](https://nixos.org/) add the snippet below to your
...
 installation , or in `~/.config/nix/nix.conf` if you have a local installation:
```
Comment (disassembler) [inline README.md]:
```suggestion
```

**Review by disassembler — CHANGES_REQUESTED** [review]
Some suggestions to add flakes and not support config in home directory

**Comment by JaredCorduan** [comment]
thank you for the suggestions @disassembler !

**Review by disassembler — APPROVED** [review]

**Pull request merged**